src/main/java/com/google/devtools/build/lib/remote/util/Utils.java

```diff
@@ -40,4 +48,51 @@ private Utils() {}
       throw new IOException(e.getCause());
     }
   }
+
+  /**
+   * Returns the (exec root relative) path of a spawn output that should be made available via
+   * {@link SpawnResult#getInMemoryOutput(ActionInput)}.
+   */
+  @Nullable
+  public static PathFragment getInMemoryOutputPath(Spawn spawn) {
```

Comment:
Where are this other new methods gonna be used? Would that be possible to place them there and not in the Utils class?

Reply:
getInMemoryOutputPath is used in `RemoteSpawnCache` and `RemoteSpawnRunner`. Does that answer your question?

Reply:
Yes, thanks! Just thought that perhaps there is a way to avoid using Utils class.